assets/app/view/game/spreadsheet.rb

```diff
@@ -177,7 +176,11 @@ def render_titles
           extra << h(:th, render_sort_link('Buying Power', :buying_power))
           extra << h(:th, render_sort_link('Interest Due', :interest))
         end
+        if (@diff_corp_sizes = @game.all_corporations.map { |c| @game.corporation_size(c) }.uniq != [:small])
```

Comment:
Just one nitpick, this iterates just once over the corporations:

```suggestion
        if (@diff_corp_sizes = @game.all_corporations.any? { |c| @game.corporation_size(c) != :small })
```